nx-dev/ui-cloud/src/lib/enhance-with-ai.tsx

```diff
@@ -10,36 +13,61 @@ import { ReactElement } from 'react';
 
 const features = [
   {
-    name: 'AI explainer',
+    name: 'Self-healing CI',
     description:
-      "Debug task errors on your CI pipeline directly in your pipeline's UI.",
-    icon: CodeBracketIcon,
+      '    Automatically fixes flaky tests and legitimate bugs in your CI pipeline, letting you merge with confidence and eliminating wasted time babysitting PRs.',
+    icon: HeartIcon,
+    isAvailable: false,
+    link: '',
+  },
+  {
+    name: 'AI data analysis',
+    description:
+      'Ask questions about your runs and tasks in natural language. Get real-time, data-driven answers to "Why did this build slow down?" or "Which tests are the flakiest this week?"',
+    icon: CircleStackIcon,
+    isAvailable: false,
+    link: '',
+  },
+  {
+    name: 'Nx MCP (Model Context Protocol)',
+    description:
+      'Gives your AI assistant the "map" it needs. It provides tools for understanding your workspace structure, enabling faster and more reliable coding, debugging, and refactoring.',
```

Comment:
something about "It provides tools" feels off to me, but again, splitting hairs right now and I'm unsure what I'd change it to.